stubs/Flask-SocketIO/flask_socketio/namespace.pyi

```diff
@@ -0,0 +1,46 @@
+from _typeshed import Incomplete
+from typing import Any, Callable, TypedDict, TypeVar
+from typing_extensions import TypeAlias
+
+from socketio import Namespace as _Namespace
+
+from .socketio import SocketIO
+
+_T = TypeVar('T')
+_Json: TypeAlias = dict[str, _Json] | list[_Json] | int | str | float | bool | None
+
+class _Acks(TypedDict):
+    args: Any
+    namespace: Any
+
+class Namespace(_Namespace):
+    acks: Acks | None
+    socketio: SocketIO | None
+
+    def __init__(self, namespace=None) -> None: ...
+    def trigger_event(self, event: Incomplete, *args): ...
+    def emit(
+        self,
+        event: Incomplete,
+        data: Incomplete | None = None,
+        room: str | None = None,
+        include_self: bool = True,
+        namespace: str | None = None,
+        callback: Callable[..., _T] | None = None,
+    ) -> _T | tuple[str, int]: ...
+    def send(
+        self,
+        data: str | list[_Json] | dict[str, _Json],
```

Comment:
`data` could also be a number or a tuple.

Reply:
Changed to `data: Any`